index.html

```diff
@@ -51,6 +54,147 @@ <h1 class="light-green">I’m a software developer</h1>
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+          <div class="project-placeholder">
+            <div class="properties">
+              <h3 class="project-title .dark-blue">
+                Multi-Post Stories Gain+Glory
+              </h3>
+              <ul class="flex-div tags">
+                <li><a href="#">Ruby on rails</a></li>
+                <li><a href="#">css</a></li>
+                <li><a href="#">JavaScript</a></li>
+                <li><a href="#">html</a></li>
+              </ul>
+              <button type="button" class="see-demo-btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="about-me">
+        <img class="aboutme-img1" src="img/aboutme-img1.png" alt="png1" />
```

Comment:
- [ ] <b>REQUIRED:</b> The project requires to use background images, not img tags. Kindly fix it.
![scrnli_3_19_2022_8-03-35 PM](https://user-images.githubusercontent.com/45217364/159133067-1358bec8-160d-4c3f-9a45-aa9d83bed1da.png)